lib/zwave/system/capabilities/alarm_tamper/NOTIFICATION.js

```diff
@@ -17,13 +20,7 @@ module.exports = {
 				return true;
 			}
 
-			if (report['Event (Parsed)'] === 'Event inactive' &&
-				report.hasOwnProperty('Event Parameter') &&
-				(report['Event Parameter'][0] === 3 ||
-					report['Event Parameter'][0] === 4 ||
-					report['Event Parameter'][0] === 9)) {
-				return false;
-			}
+			if (report['Event (Parsed)'] === 'Event inactive') return false;
```

Comment:
i don't agree on removal of the even parameters, why is better explained in an example:

motion turns on, before motion is turned off the tamper turns on also,
after the tamper cancellation it will turn off tamper, but will ALSO turn of motion even if the user has it way higher for the cancellation

the same will happen if a window/door sensor also has a motion sensor (philio and a few rebrands)